src/core/openthread.cpp

```diff
@@ -1376,6 +1376,11 @@ bool otIsDiscoverInProgress(otInstance *aInstance)
     return aInstance->mThreadNetif.GetMle().IsDiscoverInProgress();
 }
 
+ThreadError otSendMacDataRequest(otInstance *aInstance)
```

Comment:
Could you please add an empty implementation in `examples/drivers/windows/otApi/otApi.cpp` that just returns a Not Implemented error?